src/RedialContext.js

```diff
@@ -129,7 +122,7 @@ export default class RedialContext extends Component {
   }
 
   reloadComponent(component) {
-    this.load(component, this.props, true);
+    this.load([component], this.props, true);
```

Comment:
I do not think we need to do this here, `redial` should take care of this for us.